internal/services/gateway/action/org.go

```diff
@@ -180,6 +181,10 @@ type AddOrgMemberResponse struct {
 }
 
 func (h *ActionHandler) AddOrgMember(ctx context.Context, orgRef, userRef string, role cstypes.MemberRole) (*AddOrgMemberResponse, error) {
+	if h.organizationMemberAddingMode != config.OrganizationMemberAddingModeDirect && !common.IsUserAdmin(ctx) {
```

Comment:
Now you should delete existing org invitation for that member when directly adding to an org.

Reply:
@sgotti I remove existing org invitation in the configstore action